lib/job.js

```diff
@@ -133,6 +133,28 @@ class Job extends BaseModel {
     }
 
     /**
+     * Return all running builds that belong to this job
+     * @return {Promise}        List of running builds
+     */
+    getRunningBuilds() {
+        const listConfig = {
+            params: {
+                jobId: this.id,
+                status: 'RUNNING'
```

Comment:
That is true! I guess I can put RUNNING || QUEUED here but I would need to modify the datastore dynamo to support something like this.